java/src/jmri/script/Bundle.properties

```diff
@@ -1,3 +1,5 @@
 allScripts=All Script Files
-python=Python Script Files
+ECMAScript=ECMAScript Files
+python=Jython Files
+Python=Python 3 Files
```

Comment:
What is this used for? My concern is that if `Python` and `python` stands for two different things, the risk of error is high.

Or is this an error and that it should have been `jython=Jython Files` instead?